main.tf

```
@@ -59,9 +67,9 @@ resource "aws_security_group" "this" {
     content {
       description      = lookup(egress.value, "description", null)
       prefix_list_ids  = lookup(egress.value, "prefix_list_ids", null)
-      from_port        = lookup(egress.value, "from_port", null)
-      to_port          = lookup(egress.value, "to_port", null)
-      protocol         = lookup(egress.value, "protocol", null)
+      from_port        = lookup(egress.value, "from_port")
```

I believe, since the attributes are _all_ required, we can simplify the logic for each of these:

```suggestion
      from_port        = egress.value.from_port
```

---

And by "each of these" I really mean the whole "content" block, not just the three you touched so far in the pr...